src/main/java/com/iota/iri/network/protocol/Protocol.java

```diff
@@ -17,24 +17,24 @@
     /**
      * <p>
      * The supported protocol versions by this node. Bitmasks are used to denote what protocol version this node
-     * supports in its implementation. The MSB acts as a starting point. Up to 32 bytes are supported in the handshake
+     * supports in its implementation. The LSB acts as a starting point. Up to 32 bytes are supported in the handshake
      * packet, limiting the amount of supported denoted protocol versions to 256.
      * </p>
      * <p>
      * Examples:
      * </p>
      * <ul>
-     * <li>[10000000] denotes that this node supports protocol version 1.</li>
-     * <li>[11100000] denotes that this node supports protocol versions 1, 2 and 3.</li>
-     * <li>[01110110] denotes that this node supports protocol versions 2, 3, 4, 6 and 7.</li>
-     * <li>[01110110, 10001010] denotes that this node supports protocol versions 2, 3, 4, 6, 7, 9, 13 and 15.</li>
-     * <li>[01110110, 10001010, 10001000] denotes that this node supports protocol versions 2, 3, 4, 6, 7, 9, 13, 15,
+     * <li>[00000001] denotes that this node supports protocol version 1.</li>
+     * <li>[00000111] denotes that this node supports protocol versions 1, 2 and 3.</li>
+     * <li>[01101110] denotes that this node supports protocol versions 2, 3, 4, 6 and 7.</li>
+     * <li>[01101110, 01010001] denotes that this node supports protocol versions 2, 3, 4, 6, 7, 9, 13 and 15.</li>
+     * <li>[01101110, 01010001, 00010001] denotes that this node supports protocol versions 2, 3, 4, 6, 7, 9, 13, 15,
```

Comment:
May I ask what prompted to change from Big Endian to Little Endian?
It wasn't enough optimizing it for space, but you also want to optimize the handshake for CPUs?

I just find it even less readable and more confusing...

Reply:
>May I ask what prompted to change from Big Endian to Little Endian?

The endianess is still the same, I just swapped the *bits* from starting from left to start from the right. The byte order is still the same as before.

I've changed it in reverse folder because usually one shifts bits to the right and it allows us to check whether a position's bit was set by comparing it with `1` instead of `0b10000000` (which makes more sense imho)

Reply:
Yeah, it confused me because I started reading right to left, then jumping right again with my eyes (reminded me of Little Endian, but yeah I shouldn't have used that term)

I guess as long as the documentation is fine I don't care so much wither way, either though I personally rather reading left to right the entire thing at the cost of seeing `0b10000000` in the code.